schema/library.json

```diff
@@ -305,10 +305,11 @@
             "Section 6.4.3.4.2 in 90.1-2016"
         ],
         "description_datapoints": {
-            "no_of_occ": "Number of occupants",
+            "o": "Number of occupants",
             "m_oa": "Air terminal outdoor air volume flow rate",
-            "eco_onoff": "Air System Outdoor Air Economizer Status",
-            "tol": "Tolerance for the num of occupants"
+            "eco_onoff": "Air system outdoor air economizer status",
+            "tol_o": "Tolerance for the number of occupants",
+            "tol_m_oa": "Tolerance for the air terminal air volume flow rate"
```

Comment:
Edited existing text to match with new verification. Also, some data points description where missing. @leijerry888, perhaps we need a test that makes sure that this list is aligned with the point list in the verification python file?

Reply:
That's a good point. Very much needed.